### PR TITLE
Warehouse FAQ update

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Added warehouse FAQ to add table prefixes if there are multiple sources connected to the same destination.

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/destinations/warehouse-destinations/faq.mdx [hunk 1]
```diff
@@ -205,6 +205,18 @@ Refer to the <Link to ="/destinations/warehouse-destinations/">warehouse-specifi
...
+### How can I add the table prefixes if there are multiple sources connected to the same destination?
+
+You can use RudderStack's <Link to="/features/transformations/">Transformations</Link> feature to filter the events by the <Link to="/dashboard-guides/sources/#source-details">`sourceID`</Link> and then update the event name to add a prefix. 
+
+A sample transformation logic is shown below:
+
+```javascript
+if (SOURCE_ID == '<your_source_ID>') {
+  event.event = '<some_prefix>' + event.event;
+}
```
Comment (Jayachand) [inline docs/destinations/warehouse-destinations/faq.mdx:217]: lets show entire transformation
`export function transformEvent(event, metadata) {
    const meta = metadata(event);
    if (meta.sourceId === <your_source_id>) {
        event.event = '<prefix>' + event.event;
    }
    return event;
}`

File: docs/destinations/warehouse-destinations/faq.mdx [hunk 2]
```diff
@@ -205,6 +205,18 @@ Refer to the <Link to ="/destinations/warehouse-destinations/">warehouse-specifi
 If you do not set the <strong>Namespace</strong> field, RudderStack sets the source name as the schema name, with some modifications.
 </div>
 
+### How can I add the table prefixes if there are multiple sources connected to the same destination?
```
Comment (Jayachand) [inline docs/destinations/warehouse-destinations/faq.mdx:208]: `single destination` instead of `same destination` ?

Comment (ameypv-rudder) [inline docs/destinations/warehouse-destinations/faq.mdx:208]:
```suggestion
### How can I add the table prefixes if there are multiple sources connected to a single destination?
```

File: docs/destinations/warehouse-destinations/faq.mdx (hunk 1 above)
Comment (ameypv-rudder) [reply docs/destinations/warehouse-destinations/faq.mdx:221]: This is done.

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**